tests/test_helpers.py

```diff
@@ -47,6 +47,8 @@
     (["helper", "l_contacts", "run", "-n", "ny", "-i", "col", "-o", "coffee", "-d", "2019-01-15", "-r", "2"],
      ""
      ),
+    (["helper", "u_logurl", "20ly_newprojectum", "this is a test pt.2"],
```

Comment:
Sometimes the schema can be like documentation so it is better if we put something more realistic here.  It doesn't have to be a valid or full length Gdoc URL, but it should look something like it so people know what a valid url should look like.

Reply:
Yes! I did see your comment and have been working on it since yesterday. As a side note, I have finished the computation part of my 20vl_distancemetricimplementation projectum and need cif files of either extracted distance lists or PDFs of existing structures to complete the UCs. I tried reaching out in the slack channel and just asked Long a few minutes ago when he asked me to update my milestones on who might be able to send me the data but I still haven't received any word back. Do you know where/from whom I can obtain this coveted data?

Reply:
Nvm! Long just replied saying he can get me the files :D yay